### DISCUSSION TITLE
Using just the async typeahead feature?

### DISCUSSION THREAD

**Discussion by Squirreljetpack** [marlonrichert/zsh-autocomplete#596] [Q&A]
Async completion is very cool, but I have my own completion system, keybinds and init.
To clarify, currently when typing, I can browse history with autosuggestions, and up, down keys. Pressing tab brings up a menu of completions.
Is it possible to setup a part of zsh-autocomplete, so that the autocompletions are automatically updated, and no new completions are installed? Is the async updating of completions connected to everything else or is it possible to load a snippet just for that?

**Comment by marlonrichert** [comment]
Add to your `.zshrc` file _before_ sourcing Autocomplete:
```zsh
zstyle ':autocomplete:(config|key-bindings|recent-dirs)' enabled no
```

No guarantees that this will work out of the box with your own config, though. You might have to tweak it a bit.

**Reply by Squirreljetpack** [reply]
Tried zstyle ':autocomplete:(key-bindings|recent-dirs)' enabled no, works great, thank you very much. Seems config is needed for the automatic completion menu.
The only issue I noticed was that the plugin doesn't play great with zsh-autosuggest
Using this order
zi wait lucid blockf light-mode for \
  marlonrichert/zsh-autocomplete \
  atload"!_zsh_autosuggest_start" \
    zsh-users/zsh-autosuggestions
    
and setting tab to (say) Make Tab go straight to the menu and cycle there (from Readme), got the two work well in tandem mostly.

Couple of minor issues I'm facing: git completion is a bit strange (i.e. does not automatically display completion menu). Autosuggestions up-line-or-history doesn't work.
But I'm just leaving this here for documentation, I'll update if I find something
